src/Sylius/Behat/Service/Mocker/PaypalApiMockerInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Service\Mocker;
+
+/**
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
+interface PaypalApiMockerInterface
```

Comment:
What do you mean? From my point of view, the current PayPal mocker implementation is quite coupled to its usage (it just returns some responses in given order or `send` method call).

Btw. can we get rid of `MockerContainer`? Just load some naive implementations in `test` and `test_cached` environment.